src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -327,7 +327,7 @@ public function setAnalysis(Index $index, AnalysisFactoryInterface $analysis)
         $ngramDiff = 1;
         if (isset($analysers['tokenizer']) && count($analysers['tokenizer']) > 0) {
             foreach ($analysers['tokenizer'] as $tokenizer) {
-                if ($tokenizer['type'] === 'nGram') {
+                if ($tokenizer['type'] === 'ngram') {
```

Comment:
I would add a strtolower around `$tokenizer['type']` to be backwards compatible with older code, we can remove it later when only ES8+ is supported as everyone using `nGram` will have got the elasticsearch warning by then